sdk/tests/js/webgl-test-utils.js

```diff
@@ -3296,6 +3296,95 @@ function linearChannelToSRGB(value) {
     return Math.trunc(value * 255 + 0.5);
 }
 
+/**
+ * Return the named color in the specified color space.
+ * @param {string} colorName The name of the color to convert.
+ *        Supported color names are:
+ *            'Red', which is maximum sRGB red.
+ *            'Green', which is maximum sRGB green.
+ * @param {string} colorSpace The color space to convert to. Supported
+          color spaces are:
+ *            null, which is treated as sRGB
+ *            'srgb'
+ *            'display-p3'.
+ * @return {!Array.<number>} color The color in the specified color
+ *        space as an 8-bit RGBA array with unpremultiplied alpha.
+ */
+var namedColorInColorSpace = function(colorName, colorSpace) {
+  var result;
+  switch (colorSpace) {
+    case null:
+    case 'srgb':
+      switch(colorName) {
+        case 'Red':
+          return [255, 0, 0, 255];
+        case 'Green':
+          return [0, 255, 0, 255];
+          break;
+        default:
+          throw 'unexpected color name: ' + colorName;
+      };
+      break;
+    case 'display-p3':
+      switch(colorName) {
+        case 'Red':
+          return [234, 51, 35, 255];
+          break;
+        case 'Green':
+          return [117, 251, 76, 255];
```

Comment:
Similarly here? I notice this isn't just a swizzle of the display-p3 channel values for sRGB red.

Reply:
I've added a link (at the top of the function) to the section of CSS color level 4 that goes over the conversion formulas. It may be that we will end up wanting to add functions to do the conversion at some point (it depends on what we end up encountering as we add more tests that cover this functionality).

I also specified the colors involved as CSS colors (in the new syntax that only WebKit supports, but everyone will get there).

Reply:
Thanks.